src/tests/utils/compatibility-stories-factory.tsx

```diff
@@ -0,0 +1,24 @@
+import {RenderForm} from '@lib/renderer';
+import {PlayFunction} from '@storybook/csf/dist/story';
+import {Args, ComponentStory} from '@storybook/react';
+import React from 'react';
+// @ts-ignore
+import {Form} from 'react-formio';
+
+/**
+ * Generates a tuple of 2 stories, one for Form.io and one For RenderForm. Both stories receive
+ * `args` and `play` function.
+ */
+export const compatibilityStoriesFactory = (args: Args, play: PlayFunction) => {
```

Comment:
Let's make this function signature take an object with keys `args` and `play`, so that we mimick Storybooks interface. The advantage is that those names are very explicit then in the places where the factory is called.

Reply:
I don't like this approach, it makes implementations less concise. I prefer the current shape.

Reply:
Leave it as is then, but if we add support for `parameters` and `decorators` too I'm not looking forward to updating the existing stories.